tox.ini

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 88
+ignore = E203, E266, E501, W503
```

Comment:
[I just fixed a few things in the CookieCutter template.](https://github.com/getpelican/cookiecutter-pelican-plugin/commit/b83e86cc39f947ab3ee57d3d948e54298f54ae6a) Please change the ignore list to:
```
ignore = E203, W503
```